geographic_msgs/GeoPoint coordinates
float32 bearing